### PR TITLE
Linux 6.2

### PR DESCRIPTION
**Pull request by 13r0ck**
This should also enable intel arc-gpu support

### CODE REVIEW

**Comment by n3m0-22** [comment]
Tested Pop 22.04 on `gaze15`, `Dev One`, `lemp11`, `oryp4`

Most testing has passed. I am unable to install the current 470 driver.

[nvidia-470-error.txt](https://github.com/pop-os/linux/files/10845154/nvidia-470-error.txt)

I have seen the following error on the `gaze15` when running the `fwts` test.

`tpm tpm0: A TPM error (256) occurred attempting get random`

I have only seen this when running the test in hybrid graphics mode. I have not seen it suspending manually. It does not seem to affect the results as I have run `fwts s3 --s3--multiple 150` in each graphics mode with all test passing. I am currently testing to see if [NVIDIA 525.89.02](https://github.com/pop-os/nvidia-graphics-drivers/pull/178) has the same issue.

I have also tested ZFS without issue.

[linux-6.2-testing.txt](https://github.com/pop-os/linux/files/10845181/linux-6.2-testing.txt)

**Comment by n3m0-22** [comment]
In testing on Ubuntu 22.04 I am seeing some potential issues with suspend/resume. So far it has only been on machines with NVIDIA. I will update as I investigate further.

These are the steps to recreate as best as I can tell so far.

 - Set graphics mode to NVIDIA
 - run `fwts s3 --s3--multiple 150`

On the `gaze15` and `oryp4` I have seen this test freeze, but not every time. When looking at the `results.log` it shows the test stopping after the fist few times. When it gets in this state only rebooting by holding down the power button has worked for me.

**Comment by n3m0-22** [comment]
All testing is now passing on Ubuntu 22.04. 

As a side note `fwts` will need to be updated from `22.03.00` to `23.01.00`, or built from source the get accurate results.

**Comment by leviport** [comment]
With this and https://github.com/pop-os/mesa/pull/16 installed, the A770 seems to be working in major-r2. I'll try running a couple games to see how well it performs.

**Comment by leviport** [comment]
I just noticed the built-in mic on pang12 isn't working. Did we maybe drop https://github.com/pop-os/linux/pull/205 in the rebase inadvertently? Might have been dropped for 6.1.11 as well, but I'll have to double-check in a bit.

**Comment by 13r0ck** [comment]
@leviport, my bad. Thanks for noticing that. Just pushed those commits back here.

File: sound/pci/hda/patch_conexant.c [hunk 1]
```diff
@@ -1121,6 +1121,7 @@ static int patch_conexant_auto(struct hda_codec *codec)
  */
 
 static const struct hda_device_id snd_hda_id_conexant[] = {
+	HDA_CODEC_ENTRY(0x14f120d0, "CX11970", patch_conexant_auto),
```
Comment (leviport) [inline sound/pci/hda/patch_conexant.c:1124]: I think this is the same as line 1128, isn't it? I'm guessing it got added upstream at some point. We might not need this specific commit any longer.

Comment (leviport) [reply sound/pci/hda/patch_conexant.c:1124]: Actually the combo jack also stopped working properly without those commits, so I bet we still need it. I'll double-check once the build finishes.

**Comment by leviport** [comment]
Those two commits have fixed the built-in mic and combo jack detection on pang12!

**Review by XV-02 — APPROVED** [review]
Between the extensive testing by @leviport and @n3m0-22 , and confirming a lack of regressions on a desktop with discrete nvidia graphics (and checking integrated desktop graphics), this looks good to go.

**Comment by leviport** [comment]
Let's just make sure https://github.com/pop-os/xserver-xorg-video-amdgpu gets released along with it

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**